source/collections/capped-collection.txt

```diff
@@ -29,7 +29,7 @@ Procedure
 
    .. step:: Enter the collection name.
 
-   .. step:: Click the :guilabel:`Advanced Collection Options` dropdown.
+   .. step:: Click the :guilabel:`Additional preferences` dropdown.
```

Comment:
[Suggestion]

Sorry this is a little out of scope but maybe we could update the steps to not have punctuations per the [new style guide](https://www.mongodb.com/docs/meta/style-guide/information-types/prototypes/task/#steps).